llvm/include/llvm/CAS/CASDB.h

```diff
@@ -277,6 +277,7 @@ class CASDB : public CASIDContext {
 template <class HandleT> class ProxyBase : public HandleT {
 public:
   const CASDB &getCAS() const { return *CAS; }
+  CASDB &getCAS() { return *CAS; }
```

Comment:
I don't see why this is needed? If needed, a schema might just hold a mutatable CAS reference that can be used?

Might be good treat `ObjectProxy` as a readonly entity even though it doesn't matter too much if CAS is mutable or not underneath.

Reply:
(per offline discussion with Steven)

This is needed so I can do `getCAS().getProxy()` inside the proxy classes